libcrux-ml-dsa/src/simd/traits.rs

```diff
@@ -3,8 +3,8 @@ use crate::constants::{Eta, Gamma2};
 // Each field element occupies 32 bits and the size of a simd_unit is 256 bits.
 pub(crate) const COEFFICIENTS_IN_SIMD_UNIT: usize = 8;
 
-pub(crate) const SIMD_UNITS_IN_RING_ELEMENT: usize =
-    crate::constants::COEFFICIENTS_IN_RING_ELEMENT / COEFFICIENTS_IN_SIMD_UNIT;
+//COEFFICIENTS_IN_RING_ELEMENT / COEFFICIENTS_IN_SIMD_UNIT;
```

Comment:
Same here, I guess the proof can't handle this. Let's add a comment.

Reply:
added a note